nebula-logger/main/logger-engine/classes/LogEntryEventBuilder.cls

```diff
@@ -141,93 +141,263 @@ global with sharing class LogEntryEventBuilder {
     // DML-result builder methods. All* of the Result classes behave the same (*except Upsert)
     // ...but Salesforce didn't bother to use an interface for the classes, so instead - overloads!
     /**
-     * @description Sets the log entry event's DML result fields
+     * @description Sets the log entry event's database operation result fields
      * @param  deleteResult The instance of `Database.DeleteResult` to log
      * @return              The same instance of `LogEntryEventBuilder`, useful for chaining methods
      */
     global LogEntryEventBuilder setDatabaseResult(Database.DeleteResult deleteResult) {
+        if (this.shouldSave == false) {
```

Comment:
seems like these two `if` statements in each method can be combined!

Reply:
Yeah, good call, I'll consolidate the guard clauses